tnefparse/cmdline.py

```diff
@@ -97,6 +100,16 @@ def tnefparse():
             sys.stderr.write("Successfully wrote %i files\n" % len(t.attachments))
             sys.exit()
 
+        elif args.zip:
+            tfp.seek(0)
+            pth = args.path.rstrip(os.sep) + os.sep if args.path else ''
+            test = tfp.read()
+            a = to_zip(test)
```

Comment:
Unless I miss something, I think there could be a better variable name instead of "test".

Reply:
I ended up streamlining this; the whole unnecessary seeking and second reading is now gone; see added commits.